e2e/playwright/command-bar-tests.spec.ts

```diff
@@ -34,20 +36,35 @@ test.describe('Command bar tests', () => {
 
     // Click the line of code for xLine.
     await page.getByText(`close()`).click() // TODO remove this and reinstate // await topHorzSegmentClick()
-    await page.waitForTimeout(100)
 
-    await page.getByRole('button', { name: 'Extrude' }).click()
-    await page.waitForTimeout(200)
-    await page.keyboard.press('Enter')
-    await page.waitForTimeout(200)
-    await page.keyboard.press('Enter')
-    await page.waitForTimeout(200)
+    await toolbar.extrudeButton.click()
+    await cmdBar.expectState({
+      stage: 'arguments',
+      commandName: 'Extrude',
+      currentArgKey: 'sketches',
+      currentArgValue: '',
+      headerArguments: {
+        Sketches: '',
+        Length: '',
+      },
+      highlightedHeaderArg: 'sketches',
+    })
+    await cmdBar.progressCmdBar()
+    await cmdBar.progressCmdBar()
+    await cmdBar.expectState({
+      stage: 'review',
+      commandName: 'Extrude',
+      headerArguments: {
+        Sketches: '1 segment',
+        Length: '5',
+      },
+    })
+    await cmdBar.progressCmdBar()
```

Comment:
Most of the changes here and below are to account for:
1. the need to press next on selections now that we can select multiple sketches and can't have `skip: true`
2. changing "Selection" for "Sketches"

And I took the opportunity to modernize those tests a bit to new fixtures